tests/test_searchoff.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+import pytest
+from scipy.signal.windows import triang
+
+from Subfunction_Module import searchoff
+
+
+def test_searchoff():
+    """
+    Test for the "test_searchoff" function;
+    Located in the Subfunction Module of py_pecg.
+    """
+
+    #                              ___  /\  ___
+    # Modified Triangular Pulse:      \/  \/
+    Test_Signal_1 = np.concatenate(
+        (
+            np.ones(10),
+            1 - 0.5 * triang(10, sym=True),
+            1 + 1 * triang(10, sym=True),
+            1 - 0.5 * triang(10, sym=True),
+            np.ones(10),
+        )
+    )
```

Comment:
This section is a duplicate with `test_searchon`. I'd make it a [pytest fixture](https://docs.pytest.org/en/6.2.x/fixture.html) to avoid code duplication